src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -327,11 +328,21 @@ VtolAttitudeControl::Run()
 		// check in which mode we are in and call mode specific functions
 		switch (_vtol_type->get_mode()) {
 		case mode::TRANSITION_TO_FW:
+			// vehicle is doing a transition to FW
+			_vtol_vehicle_status.vehicle_vtol_state = vtol_vehicle_status_s::VEHICLE_VTOL_STATE_TRANSITION_TO_FW;
+
+			_fw_virtual_att_sp_sub.update(&_fw_virtual_att_sp);
```

Comment:
@sfuhrer Why do you need to update here again, don't you update just before the switch?

Reply:
No likely not, good find!

Reply:
Removed it, see last commit